index.html

```diff
@@ -296,9 +301,35 @@ <h3>Skills</h3>
             </div>
             </div>
             </section>
-        
-    
+            <section id="contact-form">
+                          
+                   <form id="form-1" action="https://formspree.io/f/xyyobakd" method="post">
```

Comment:
- [ ] Kindly fix the about and contact section with margin and padding check Figma design [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
- Your Design
![Captura](https://user-images.githubusercontent.com/81444338/171872817-b8344902-a271-4b58-8854-33627def7c3d.PNG)
- Figma Design
![Capturaf](https://user-images.githubusercontent.com/81444338/171872898-9eba2174-3aab-4098-bcd9-3a24d1d479fc.PNG)

Reply:
Done as advised

Reply:
Your current application is still not looking as requested by the previous reviewer. Here is the current state of your application. Kindly update, and then ask for rereview.
![image](https://user-images.githubusercontent.com/33268927/171994537-9dfb80c1-d2e9-409d-b6bc-e03c3d39cc46.png)

Reply:
I am, not sure what do you mean, if you mean the space between the left and the right section is not matching , then i don't think , it is mandatory requirement

Reply:
Done as advised